### ISSUE TITLE
Multiple quick deposit buttons/ borrow buttons- as well as inactive tooltips

### ISSUE THREAD

**Issue by TechsUsInc** [bitshares/bitshares-ui#2130]
**Describe the bug**
Open.xxx and bit.xxx assets regardless of pairing to BTC/BTS/USD/CNY display inconsistent, inapplicable, multiple, and sometimes missing quick deposit and borrow buttons. In some pairs, mostly borrow, the tooltip is inactive.

**To Reproduce**
1. Go to exchange.
2. Go to Open.BTC : BTS pair by clicking on BTC market, then BTS.
3. Note the multiple quick deposit buttons.
4. Switch pairs, to Open.BTC : ETH by clicking on ETH market, then Open.BTC.
5. Note there are now 4 quick deposit buttons.
6. Switch pairs, to Open.BTC : BTS and then again to Open.BTC : USD again noting the multiple quick 
   deposit buttons.
7. I found the same to be true for most other Open.xxx and bit.xxx assets pairing.

![181031b-bit cny-capture](https://user-images.githubusercontent.com/18186497/48132745-2fb31680-e25a-11e8-8183-77478e901f48.JPG)
![181031b-open btc-capture](https://user-images.githubusercontent.com/18186497/48132746-2fb31680-e25a-11e8-98b9-67d6ae6878d4.JPG)
![181031b-open eth-capture](https://user-images.githubusercontent.com/18186497/48132747-2fb31680-e25a-11e8-8ba2-e5bccbdbca36.JPG)

**Comment by gibbsfromncis** [comment]
Also reproducible if just open link - https://wallet.bitshares.org/#/market/BTS_OPEN.ETH

**Comment by startailcoon** [comment]
The reason we have two buttons is because one is for Open Ledger "Deposit" and the other one is for Tradeblock "Buy".  

As we have a new component for depositing trough bridges we should move to the new one and retire the old code that was specific for Open Ledger.

### Suggestions to solve this issue
- Remove "Quick Deposit" for the deposit feature, as it's available in the menu
- Use Quick Deposit for the Tradeblock Buy feature.

Suggestions or inputs?

**Comment by sschiessl-bcp** [comment]
I personally like the deposit feature there. I would suggest to think of proper UX.

It could be a e.g. that if there are two choices the button opens a select menu

**Comment by startailcoon** [comment]
Could do a drop-out selection perhaps, if we have more than one type.

If we have both I suggest we display them as follows

- Deposit through Gateway
- Buy through Bridge

**Comment by sschiessl-bcp** [comment]
@gibbsfromncis

Does ANT provide a new way to do tooltips? ReactTooltips seems to be unreliable

**Comment by startailcoon** [comment]
Yes, it does. And we also have the pop-out function in the style guide.

**Comment by startailcoon** [comment]
## Preview
![bitshares-quickdeposit](https://user-images.githubusercontent.com/12114550/48313314-afbee080-e5ba-11e8-98f1-68d1deaf2cde.gif)

If we have miltiple choices, the user is presented with them on clicking the Quick Deposit button.

**Comment by sschiessl-bcp** [comment]
That looks suoer!

My 2 cents:

Please keep the tooltip for all three buttons.

Quick deposit: say which asset and that on click the sector opens 

Bridge/Gateway: say which asset and what a gateway/bridge is

Add a title line in the pop-up that mentions the asset 

If there is one button show the respective tooltip gateway/bridge

**Comment by startailcoon** [comment]
Updated with tooltips

Popover does not handle counterpart.translate, so it's using the Popover component to show the tooltips as well.

![bitshares-quickdeposit_3](https://user-images.githubusercontent.com/12114550/48316902-e95b1000-e5e9-11e8-8ce5-20556935dd2c.gif)
